packages/legend-server-sdlc/src/models/configuration/ProjectConfiguration.ts

```diff
@@ -35,15 +45,19 @@ export class ProjectConfiguration implements Hashable {
   groupId!: string;
   artifactId!: string;
   projectStructureVersion!: ProjectStructureVersion;
+  platformConfigurations: ProjectServerPlatform[] | null = null;
```

Comment:
This is something weird with SDLC, but basically, we could keep these as `undefined`, so `platformConfigurations?: PlatformConfiguration[] | undefined`

we will convert `undefined` to `null` during (de)serialization